R-packages/covidcast/R/covidcast.R

```diff
@@ -301,13 +301,16 @@ summary.covidcast_signal = function(object, ...) {
 #'
 #' @details The argument structure is just as in `covidcast_signal()`, except
 #'   the first four arguments `data_source`, `signal`, `start_day`, `end_day`
-#'   are permitted to be vectors. The returned object is a list of
-#'   `covidcast_signal` data frames, of length `N = max(length(data_source),
-#'   length(signal))`. The first two arguments `data_source`, `signal` are
+#'   are permitted to be vectors. The first two arguments `data_source`, `signal` are
 #'   recycled appropriately, in the calls to `covidcast_signal()`; see example
 #'   below. The next two arguments `start_day`, `end_day`, unless `NULL`, must
 #'   be either length 1 or N.
```

Comment:
This might be a bit confusing because `N` hasn't been defined in the details section (since you cut the line where it's defined). Though it is defined in the return section; and IIRC, the details are always printed after the return paragraph?

Reply:
Right, in pkgdown you get this:

<img width="864" alt="Screen Shot 2020-11-01 at 4 14 38 PM" src="https://user-images.githubusercontent.com/711629/97815625-58a9ee80-1c5d-11eb-9bfc-56258a70d6da.png">